### DISCUSSION TITLE
为什么不能像hmcl那样在已安装版本中修改forge版本这些？

### DISCUSSION THREAD

**Discussion by Moxian-debug** [Meloong-Git/PCL#3760] [提问] [answered]
有些mod没写需要的forge版本，需要一个一个试，就要删掉整个版本重新安装，好麻烦
![java_162](https://github.com/Hex-Dragon/PCL2/assets/125735185/412c574b-54c6-49d5-8ae2-652173343118)

**Comment by MoYuan-CN** [comment, marked as answer]
#220，请移步 #227 投票

**Comment by allMagicNB** [comment]
#220（#227）
